doc/developer-notes.md

```diff
@@ -81,6 +81,10 @@ tool to clean up patches automatically before submission.
     function declarations over multiple lines using the Clang Format
     [AlignAfterOpenBracket](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)
     style option.
+  - Use a named cast or functional cast, not a C-Style cast. When casting
+    between integer types, use functional casts such as `int(x)` or `int{x}`
+    instead of `(int) x`. When casting between more complex types, use static_cast.
+    Use reinterpret_cast and const_cast as appropriate.
```

Comment:
You got the alignment wrong - there is one extra space at start of each line.

Reply:
I don't think so... Should be preceded by four spaces right?

Reply:
I see what you mean now. I think I've moved it into the correct place now